scripts/_common.sh

```diff
@@ -33,7 +33,7 @@ waitforservice() {
 
 installnode(){
 
-  if [ $DEBIAN_ISSUE ==  "stretch"]; then
+  if [ $DEBIAN_ISSUE ==  "stretch" ]; then
```

Comment:
there was also an additional space before `"stretch"` I fixed this all on master now. see Yunohost-Apps/rocketchat_ynh#55